lib/msal-node/src/client/ClientAssertion.ts

```diff
@@ -85,4 +95,16 @@ export class ClientAssertion {
     private isExpired(): boolean {
         return this.expirationTime < TimeUtils.nowSeconds();
     }
+
+    public static parseCertificate(publicCertificate: string): Array<string> {
+        const re = /-----BEGIN CERTIFICATE-----\n(.+?)\n-----END CERTIFICATE-----/gm;
+        const certs = Array<string>();
+
+        let matches;
+        while ((matches = re.exec(publicCertificate)) !== null) {
```

Comment:
Some comments for this code would be helpful.

Reply:
Added some comments.